server/utils/fieldValidation.js

```diff
@@ -1,31 +1,34 @@
 const baseJoi = require('joi')
 const dateExtend = require('joi-date-extensions')
-const postcodeExtend = require('joi-postcode')
 const moment = require('moment')
 const { getFieldName, getFieldDetail, mergeWithRight, getIn, isNilOrEmpty } = require('../utils/functionalHelpers')
 
-const joi = baseJoi.extend(dateExtend).extend(postcodeExtend)
+const joi = baseJoi.extend(dateExtend)
+
+function mapJoiErrors(joiErrors, fieldsConfig) {
```

Comment:
Pulled out this function so I could more easily use joi and map its output on the new page